### PR TITLE
Xylix's Greatest Prank

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
## About The Pull Request

Makes all the Jester's stats randomized

## Why It's Good For The Game

Because it would be kind of funny

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/peasants/jester.dm [hunk 1]
```diff
@@ -47,7 +47,11 @@
 		H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/music, pick(1,2), TRUE)
 		H.mind.AddSpell(new /obj/effect/proc_holder/spell/invoked/mockery) // Mock people to your heart's content!
+		H.STASTR = rand(1, 20)
```
Comment (1N7Slayer) [inline code/modules/jobs/job_types/roguetown/peasants/jester.dm:50]: 1,15 on all these ranges or i would not even support this ironically, pls

Comment (BadAtThisGame302) [reply code/modules/jobs/job_types/roguetown/peasants/jester.dm]: Done.

**Comment by ThePainkiller** [comment]
No, sorry but no. **Specially** combat stats like STR, END, CON and PER. This will only end up in horribly overpowered jesters.

**Comment by sanshoom** [comment]
> No, sorry but no. **Specially** combat stats like STR, END, CON and PER. This will only end up in horribly overpowered jesters.

overpowered jesters… me when i stab them in the mouth and cut their head off because they can’t afford armor or weapons. or friends.

**Comment by ThePainkiller** [comment]
Me when the jester with 15 STR and 15 Fortune just spams eyestab dagger and crits me, instantly killing me. No is no.

**Comment by sanshoom** [comment]
> Me when the jester with 15 STR and 15 Fortune just spams eyestab dagger and crits me, instantly killing me. No is no.

fortune doesn’t do anything.

**Comment by Optimism333** [comment]
Having jester even have a CHANCE to roll higher strength than the Captain is silly, and not in a funny way. It's just absurd.

Would much rather jesters were given more content/items for pranks. Complete stat randomization is low hanging fruit imo, and a nightmare for balance at the current values.

**Comment by Stutternov** [comment]
> fortune doesn’t do anything.

It does, code-diving I found out I was gaslit and wrong awhile ago.

Fortune assigns good.luck and bad.luck modifiers which, for example, good.luck assigns +10 damage on crit and stuff.

**Comment by ThePainkiller** [comment]
No updates, no way in hell to even get a chance at having barb level STR or massive stats, or guaranteed fortune bonus critchance, meme PR, let's clean up the git a bit.

**Pull request closed without merging**